tools/psa/build_tfm.py

```diff
@@ -142,15 +162,16 @@ def check_and_clone_repo(name, deps):
         logger.info('%s repo exists, checking git version...', name)
         check_repo_version(name, deps)
 
-def clone_tfm_repo():
+def clone_tfm_repo(commit):
     """
     Clone TF-M git repos and it's dependencies
     """
     check_and_clone_repo('trusted-firmware-m', dependencies)
     check_and_clone_repo('mbedtls', dependencies)
     check_and_clone_repo('mbed-crypto', dependencies)
     check_and_clone_repo('CMSIS_5', dependencies)
-    detect_and_write_tfm_version(join(TF_M_BUILD_DIR, 'trusted-firmware-m'))
+    detect_and_write_tfm_version(join(TF_M_BUILD_DIR, 'trusted-firmware-m'),
+                                  commit)
```

Comment:
We need to remove a space before `commit` to align with `join`.

Reply:
Fixed